### ISSUE TITLE
Model class API / predict / gradient update

### ISSUE THREAD

**Issue by johngrabner** [tensorflow/tensorflow#37066]
## URL(s) with the issue:
https://keras.io/models/model/
and 
https://www.tensorflow.org/api_docs/python/tf/keras/Model#predict

## Description of issue (what needs changing):
Documentation for model predict says "batch_size: Integer or None. Number of samples per gradient update.", but unless I'm missing something, there are no gradient updates while predicting.

**Comment by gowthamkpr** [comment]
Thanks for the report @johngrabner .

**Comment by yhliang2018** [comment]
Thanks! This is already updated in the [nightly version](https://www.tensorflow.org/api_docs/python/tf/keras/Model?version=nightly#predict).
